### PR TITLE
Update ccc discussion to CSR

### PR DESCRIPTION
**Pull request by jddarcy**
---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

### Reviewers
 * Jesper Wilhelmsson ([jwilhelm](@JesperIRL) - **Reviewer**) ⚠️ Review applies to d116c4020e7e5757a926480b3ec6c799f5c4e494

### Download
`$ git fetch https://git.openjdk.java.net/guide pull/19/head:pull/19`
`$ git checkout pull/19`

### CODE REVIEW

File: src/changePlanning.md [hunk 1]
```diff
@@ -74,12 +74,11 @@ also imply "implementation for an enhancement".
...
-    Y   Submission of a CSR request is required. Work may begin after the
-        request has been [accepted](glossary.html#accepted) by the CSR; the
-        changeset implementing the fix may be committed to a team forest
-        after the request has been [approved](glossary.html#approved) by the CSR.
-        A description of the CSR requirements may be found in
-        [Review Bodies](reviewBodies.html).
+    Y   Submission of a CSR request is required. The CSR must be [approved](glossary.html#approved) 
+        before the bug fix is pushed to a feature release or update release
+        repository. The work may begin concurrently with the CSR review, but may
+        need to be update in response to CSR review. A summary of the CSR process
```
Comment (JesperIRL) [inline src/changePlanning.md:80]: "update" -> "updated"

Comment (jddarcy) [reply src/changePlanning.md]: Thanks for catching that; will push with slight wording adjustment.

**Pull request closed without merging**